api/src/main/java/com/microsoft/gctoolkit/aggregator/Aggregator.java

```diff
@@ -121,10 +121,8 @@ public void onCompletion(Runnable task) {
      * Call a callback when aggregation is completed.
      */
     private void complete() {
-        Runnable t = completionTask;
-        this.completionTask = null;
-        if (t != null)
-            Executors.newSingleThreadExecutor().execute(t);
+        if (completionTask != null)
```

Comment:
The reason for this change is that it is possible for an Aggregator to get events from more than one EventSource. If the completionTask is nulled out, then the Phaser only gets decremented by the first EventSource but not the subsequent EventSources. 

The test CaptureJVMTerminationEventTest was hanging because of this issue.